frontend/src/composables/WorkSpace/useDomainData.js

```diff
@@ -101,12 +98,8 @@ export default function useDomainData() {
 	};
 
 	const addTagtoDomain = async (tag) => {
-		try {
 			await API.post(`/domain/${choosenNodeID.value}/tag/${tag.id}`);
 			refreshDomainTag(choosenNodeID.value);
-		} catch (error) {
-			console.log(err);
```

Comment:
Could be good to add error handling later on! What is happening for now if the API call is in error? Should we write a US to handle this error?

Reply:
Yes all the API calls will be handled globally on the axios middleware